docs/data_structures/transactions.md

```diff
@@ -24,4 +24,27 @@ data sources.
 | DKB  | Checking account, webpage  | Buchungstag, Wertstellung, Buchungstext, Auftraggeber / Begünstigter, Verwendungszweck, Kontonummer, BLZ, Betrag (EUR), Gläubiger-ID, Mandatsreferenz, Kundenreferenz |
 | DKB  | Credit card, webpage       | Umsatz abgerechnet und nicht im Saldo enthalten, Wertstellung, Belegdatum, Beschreibung, Betrag (EUR), Ursprünglicher Betrag  |
 | Consorsbank  | Checking account, webpage | Buchung, Valuta, Sender / Empfänger, IBAN / Konto-Nr., BIC / BLZ, Buchungstext, Verwendungszweck, Betrag in EUR |
-| MoneyMoney  | Banking application | Datum, Wertstellung, Kategorie, Name, Verwendungszweck, Konto, Bank, Betrag, Währung  |
\ No newline at end of file
+| MoneyMoney  | Banking application | Datum, Wertstellung, Kategorie, Name, Verwendungszweck, Konto, Bank, Betrag, Währung  |
+
+## Columns
+
+These are the columns contained in the main data structure
+of the transactions.
+
+| Column | Description |
+|--------|-------------|
+| `time` | Time when the transaction happened, format: unix timestamp |
+| `sender` | String containing the name of the sender of the transaction |
+| `sender_account` | String containing the account information of the sender |
+| `receiver` | String containing the name of the receiver of the transaction |
+| `receiver_account` | String containing the account information of the receiver |
+| `text` | Text of the transaction |
+| `amount` | Amount of money that has been transferred |
+| `currency` | Currency of the transaction like "EUR" or "USD" |
+
+
+**Notes**:
+
+* The unix timestamp `time` is universal since it can be used to quickly calculate differences and since it's timezone is well defined (i.e. UTC)
```

Comment:
* the incoming data is probably only dates (relative to a timezone) but without explicit timezone - in order to convert it to unixtime we would have to assume or ask for a timezone. I would prefer if the software just goes with whatever timezone it is fed with. 
* the incoming data does not have a timestamp but only days - since the time is actually not really well defined in banking accounts (transactions take a while - you can rarely guarantee an hour at which the transaction will take place etc), we would have to make assumptions here
* it's not easily readable, making comparison between input data and database entry harder at an early stage of the project
* I don't see that calculating time differences is a very frequent or very expensive task
* if we need time differences, it will probably be more like "how many workdays are between X and Y", localized stuff, that will require 3rd party code anyways
* we throw away the original raw data, replacing it by newly formatted data while making assumptions about information that is actually not really important